### PR TITLE
reduce unnecessary tests

### PR DESCRIPTION
**Pull request by VincyZhang**
## Type of Change

(1) remove unnecessary tests
(2) use local image to reduce build image cost

### CODE REVIEW

**Comment by VincyZhang** [comment]
(1) remove bert model tests
(2) remove tensorflow related unit test, ut cost from 60mins to 20mins each
(3) use local image (if exist) for chatbot workflow, reduce 20mins each

**Review by hshen14 — APPROVED** [review]

**Comment by hshen14** [comment]
@VincyZhang PR looks good, while the coverage drops. I understood the PR is to optimize the test time and reuse the docker, why the coverage drops?

File: .github/workflows/optimize-test.yml [hunk 1]
```diff
@@ -1,16 +1,15 @@
 name: Optimize Model Test
 
 on:
```
Comment (hshen14) [inline .github/workflows/optimize-test.yml:3]: do we need to keep those commented lines?

**Comment by VincyZhang** [comment]
> @VincyZhang PR looks good, while the coverage drops. Is it expected? If yes, we may need to figure out a way to bring the coverage back in another PR.

coverage drop is because we exclude tf related UT, we will test them in nightly.

**Comment by VincyZhang** [comment]
> > @VincyZhang PR looks good, while the coverage drops. Is it expected? If yes, we may need to figure out a way to bring the coverage back in another PR.
> 
> coverage drop is because we exclude tf related UT, we will test them in nightly test.

File: .github/workflows/optimize-test.yml (hunk 1 above)
Comment (VincyZhang) [reply .github/workflows/optimize-test.yml:3]: we can remove them.

**Pull request merged**